engines/jsonengine.py

```diff
@@ -117,6 +116,14 @@ def table_exists(self, dbname, tablename):
         tablename = self.table_name(name=tablename, dbname=dbname)
         return os.path.exists(tablename)
 
+    def to_csv(self):
+        """Export table from json engine to CSV file"""
+        keys = [columnname[0] for columnname in self.table.columns]
+        filename =self.table_name()
+        csv_outfile =json2csv(str(filename),header_values=keys)
```

Comment:
`filename` isn't already a string? Just wondering why we need the `str(filename)` call.

Missing spaces after `=` and `,`